.github/workflows/ci-plugin.yml

```
@@ -9,4 +9,7 @@ on:
 
 jobs:
   test:
-    uses: hapijs/.github/.github/workflows/ci-plugin.yml@master
+    uses: hapijs/.github/.github/workflows/ci-plugin.yml@workflow-min-hapi-version
```

Once https://github.com/hapijs/.github/pull/23 lands:
```suggestion
    uses: hapijs/.github/.github/workflows/ci-plugin.yml@master
```